conformance_tests/sysman/test_sysman_device/src/test_sysman_device.cpp

```diff
@@ -111,12 +111,20 @@ static void run_child_process(const std::string &device_hierarchy) {
 TEST_F(
     SYSMAN_DEVICE_TEST,
     GivenDeviceHierarchyModeCombinedWhenSysmanDeviceUUIDsAreRetrievedThenSysmanAndCoreDeviceUUIDsAreMatched) {
+  if (strcmp(getenv("ZES_ENABLE_SYSMAN"), "1") == 0) {
+    static char sys_env[] = "ZES_ENABLE_SYSMAN=0";
+    putenv(sys_env);
+  }
   run_child_process("COMBINED");
 }
 
 TEST_F(
     SYSMAN_DEVICE_TEST,
     GivenDeviceHierarchyModeCompositeWhenSysmanDeviceUUIDsAreRetrievedThenSysmanAndCoreDeviceUUIDsAreMatched) {
+  if (strcmp(getenv("ZES_ENABLE_SYSMAN"), "1") == 0) {
```

Comment:
Looks like ZES_ENABLE_SYSMAN=0 is pre-requisite for this test, please add it to the test name and also skip the test if the pre-requisite is not met.

May be a different fixture which disabled ZES_ENABLE_SYSMAN is better.

Reply:
Just thinking that ZES_ENABLE_SYSMAN=0 is mandatory for this test as we use both core & sysman. Skipping the test may fail to test this scenario in Windows OS (as ZES_ENABLE_SYSMAN is set to 1 by default when driver is installed)

Reply:
> Just thinking that ZES_ENABLE_SYSMAN=0 is mandatory for this test as we use both core & sysman. Skipping the test may fail to test this scenario in Windows OS (as ZES_ENABLE_SYSMAN is set to 1 by default when driver is installed). your thoughts on this ?

This behaviour sounds like it is implementation specific. Means the spec is not mandating "ZES_ENABLE_SYSMAN=0" . Please confirm.

In that case, we need to have test for both ZES_ENABLE_SYSMAN=0 and ZES_ENABLE_SYSMAN=1 and the ZES_ENABLE_SYSMAN=1 case will always fail (as expected for intel implementation)

Reply:
Modified as per the discussion we had offline!